src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -501,8 +501,14 @@ public void moveToTableStage(
       // get
       InputStream file;
       try {
+        logInfo(
+            "Downloading file: {} from stage: {} to move to table stage: {}",
+            name,
+            stageName,
+            tableName);
```

Comment:
Let's try to keep the change small, all the logs related to file moving/deletion could be removed. If we want to look at this particular issue, Pfizer has everything as well